### PR TITLE
main/nodejs: upgrade to 8.9.2

### PR DESCRIPTION
**Pull request by timbru31**

### CODE REVIEW

File: main/nodejs/APKBUILD [hunk 1]
```diff
@@ -5,16 +5,10 @@
 # Contributor: Tadahisa Kamijo <kamijin@live.jp>
 # Maintainer: Jakub Jirutka <jakub@jirutka.cz>
 #
-# secfixes:
-#   6.11.1-r0:
-#     - CVE-2017-1000381
-#   6.11.5-r0:
-#     - CVE-2017-14919
-#
```
Comment (jirutka) [inline main/nodejs/APKBUILD]: Why did you remove secfixes?

**Comment by timbru31** [comment]
IMO they are no longer relevant, since they are for v6, but Node.js LTS is already at v8.
If you want, I can restore it.

**Comment by jirutka** [comment]
They are relevant, we don't remove old records for secfixes.

**Comment by timbru31** [comment]
Restored.

**Pull request closed without merging**